Week_6/homework/backend/routes/todo.js

```diff
@@ -1,6 +1,10 @@
 /** Module for handling users */
+const firebase = require("../firebase/cred.js");
 const express = require("express");
-const cors = require("cors");
+const app = express()
+const cors = require('cors')
```

Comment:
You should be able to not have to call cors in the todo.js and instead call it for index.js which will automatically apply cors to all your routes. Test this though and let me know if it works